kai/cache.py

```diff
@@ -234,32 +232,18 @@ def __init__(self, task: Task, request_type: str = "llm_request"):
 
     def _path_with_limit(self, root: Optional[Task], path: Path) -> Path:
         if len(str(path)) > self._limit and root is not None:
-            root_path = Path(root.__class__.__name__)
-            stem = Path(f"depth_{self.task.depth}") / self.task.__class__.__name__
-            if isinstance(root, ValidationError):
-                root_path /= self._clean_filename(root.file)
-            if isinstance(self.task, ValidationError):
-                if not isinstance(root, ValidationError) or (
-                    isinstance(root, ValidationError) and root.file != self.task.file
-                ):
-                    stem /= self._clean_filename(self.task.file)
-            if isinstance(self.task, AnalyzerRuleViolation):
-                stem /= self.task.violation.id
-            return root_path / stem / path.name
+            root_path = root.get_cache_path(Path("."))
+            task_path = self.task.get_cache_path(Path("."))
+            print(task_path)
```

Comment:
Should probably be a send to a logger to avoid stdio collision stuff or removed

Reply:
good catch leftover from testing, sorry